src/main/java/com/google/sps/servlets/MentorApprovalServlet.java

```diff
@@ -47,14 +47,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int userId = Utility.getUserId();
     
     // Set default variables to create MentorEvidence object.
-    boolean isApprover = false;
     String mentorUsername = "";
     boolean isApproved = false;
     boolean isRejected = false;
     String paragraph = "";
+    boolean[] approver = {false, false}; // First index verifies approver as assigned to mentor and
+                                         // second checks if approver has already reviewed.
+
```

Comment:
A better name for this might be approverStatus. Right now "approver" is kind of confusing and vague when you see how it is used throughout the rest of this function.

Reply:
Done!